css/main.css

```diff
@@ -1,3 +1,472 @@
 /* Add your CSS here */
+@import url("https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0");
+
+* {
+    padding: 0%;
+    margin: 0%;
+    box-sizing: border-box;
+}
+
+:root {
+    --font-fallback: sans-serif;
+    --section-color: #E6DACD;
+    --card-color: #F4ECE6;
+    --btn-color: #0150FD;
+    --icon-color: #000;
+    --bg-color: #FFF;
+    --font1 : 'Poppins', sans-serif;
+    --font2 : 'Quicksand', sans-serif;
+    --font_3: normal normal normal 17px/1.4em avenir-lt-w01_35-light1475496,avenir-lt-w05_35-light,sans-serif;
+    
+}
+
+/* Do not forget to link this file to your HTML in the <head> */
+body {
+    font-family: var(--font2);
+    font-weight: 300;
+    font-size: 62.5%;
+}
+/* mobile design for About me page */
+
+.container {
+    display: grid;
+    grid-template-areas:
+        "header header"
+        "main main"
+        "footer footer";
+    grid-template-rows: 50px 1fr 1fr;
+    grid-template-columns: 1fr 1fr;
+    margin: 10% 0 2% 0;
+    background-color: var(--bg-color);
+    height: 155vh;
+}
+
+header {
+    grid-area: header;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 1;
+    grid-row-end: 2;
+    display:flex;
+    justify-content: space-between;
+    height: 10vh;
+    margin: 0 5% 0 6%;
+}
+
+span {
+    color: #0150FD;
+}
+
+header h1, .front h2, article>h2{
+    font-family: var(--font1);
+    font-weight: bold;
+    font-size: 1.5rem;
+}
+.forward-slash{
+    display: none;
+}
+
+header p{
+    font-family: var(--font2);
+    font-weight: lighter;
+    font-size: 1rem;
+}
+
+.nav-toggle {
+    color: #0150FD;
+    border: transparent;
+    background: transparent;
+    font-size: 1.5rem;
+}
+
+nav {
+    display: none;
+}
+main{
+    grid-area: main;
+    grid-column-start: 1;
+    grid-column-end: 3;
+    grid-row-start: 2;
+    grid-row-end: 3;
+    height: 110vh;
+}
+.first-section{
+position: relative;
+  width: auto;
+  height: 60vh;
+}
+.back{
+position: absolute;
+background: linear-gradient(0deg, #E6DACD 60%, white 40%);
+  width: 100%;
+  height: 60vh;
+}
+.front{
+position: absolute;
+  left: 5%;
+  right: 3%;
+  background-color: var(--card-color);
+  width: 93%;
+  height: 50vh;
+  top: 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
+  box-shadow: -10px 5px 50px -15px grey;
```

Comment:
if you add `height: auto` to your `front` class this should help with ensuring your image stays within your beige card -it will make sure the card is fitting all it's content within it, so the image doesn't pop out of the top